building_blocks/recipes.lua

```diff
@@ -42,46 +42,48 @@ minetest.register_craft({
 	output = 'building_blocks:fakegrass 2',
 	recipe = {
 		{'group:leaves'},
-		{"default:dirt"},
+		{homedecor.materials.dirt},
 	}
 })
 minetest.register_craft({
 	output = 'building_blocks:Fireplace 1',
 	recipe = {
-		{"default:steel_ingot", "building_blocks:sticks", "default:steel_ingot"},
+		{homedecor.materials.steel_ingot, "building_blocks:sticks", homedecor.materials.steel_ingot},
 	}
 })
 minetest.register_craft({
 	output = 'building_blocks:grate 1',
 	recipe = {
-		{"default:steel_ingot", "default:steel_ingot"},
-		{"default:glass", "default:glass"},
+		{homedecor.materials.steel_ingot, homedecor.materials.steel_ingot},
+		{homedecor.materials.glass_block, homedecor.materials.glass_block},
 	}
 })
 
 if not minetest.get_modpath("moreblocks") or not minetest.get_modpath("gloopblocks") then
 	minetest.register_craft({
 		output = 'building_blocks:gravel_spread 4',
 		recipe = {
-			{"default:gravel", "default:gravel", "default:gravel"},
+			{homedecor.materials.gravel, homedecor.materials.gravel, homedecor.materials.gravel},
 		}
 	})
 end
 
-minetest.register_craft({
-	output = 'building_blocks:hardwood 2',
-	recipe = {
-		{"default:wood", "default:junglewood"},
-		{"default:junglewood", "default:wood"},
-	}
-})
-minetest.register_craft({
-	output = 'building_blocks:hardwood 2',
-	recipe = {
-		{"default:junglewood", "default:wood"},
-		{"default:wood", "default:junglewood"},
-	}
-})
+if minetest.get_modpath("default") then
+	minetest.register_craft({
+		output = 'building_blocks:hardwood 2',
```

Comment:
here and elsewhere: it's best to use consistent parentheses; double quotes seem to be winning for lua code styling. not a blocking issue though.

Reply:
this isnt a style pr, homedecor is horrible. it does "" vs '' all over the place, as well as `groups = { group = value }` or `groups={group=value}` all over the place